src/TimeGutter.js

```diff
@@ -37,9 +39,9 @@ export default class TimeGutter extends Component {
 
   render() {
     const { resource, components, getters } = this.props
-
+    const { timeGutterWrapper: Wrapper = FallbackWrapper } = components
     return (
-      <div className="rbc-time-gutter rbc-time-column">
+      <Wrapper className="rbc-time-gutter rbc-time-column">
         {this.slotMetrics.groups.map((grp, idx) => {
```

Comment:
We can pass this.slotMetrics as an argument to the wrapper to make it more powerful. A good use case for this might be for a wrapper to render "multiple" TimeSlotGroup to support multi-timezone time view.